libs/dataset-explorer/src/lib/getDistributionBalanceMeasuresChart.ts

```diff
@@ -0,0 +1,261 @@
+    measureLocalization.KLDivergence.Name,
+    {
+      Description: measureLocalization.KLDivergence.Description,
+      KeyName: nameof<IDistributionBalanceMeasures>("KLDivergence")
+    }
+  ],
+  [
+    measureLocalization.TotalVariationDistance.Name,
+    {
+      Description: measureLocalization.TotalVariationDistance.Description,
+      KeyName: nameof<IDistributionBalanceMeasures>("TotalVarianceDist")
+    }
+  ],
+  [
+    measureLocalization.WassersteinDistance.Name,
+    {
+      Description: measureLocalization.WassersteinDistance.Description,
+      KeyName: nameof<IDistributionBalanceMeasures>("WassersteinDist")
+    }
+  ]
+]);
+
+export function getDistributionBalanceMeasuresChart(
+  distributionBalanceMeasures: IDistributionBalanceMeasures
+): IHighchartsConfig {
+  if (
+    distributionBalanceMeasures === undefined ||
+    Object.keys(distributionBalanceMeasures).length === 0
+  ) {
+    return {};
+  }
+
+  const chartLocalization =
+    localization.ModelAssessment.DataBalance.DistributionBalanceMeasures.Chart;
+  const infoIcon = "&#9432;";
+
+  const features = Object.keys(distributionBalanceMeasures);
+
+  // Since each measure has its own range of values, it can be its own subplot and
+  // therefore it has its own series, x-axis, and y-axis.
+  const multipleSeries: SeriesOptionsType[] = [],
+    xAxes: XAxisOptions[] = [],
+    yAxes: YAxisOptions[] = [];
+
+  // Keeps track of charts that are hidden at the start. Gets updated during legend click event.
+  // Since there are many subplots, start with some hidden and some visible.
+  const hiddenCharts = new Set([
+    measureLocalization.ChiSquarePValue.Name,
+    measureLocalization.KLDivergence.Name,
+    measureLocalization.CrossEntropy.Name,
+    measureLocalization.TotalVariationDistance.Name
+  ]);
+
+  // Calculate the width of each subplot and the padding between each subplot
+  const numVisibleCharts =
+    DistributionBalanceMeasuresMap.size - hiddenCharts.size;
+  const width = 100 / numVisibleCharts;
+  const padding = 5;
+
+  // Represents axis.left for each subplot, meaning at which point from the left to start the subplot
+  let axisLeftStart = 0;
+
+  // Populate the multiple series, x-axes, and y-axes
+  [...DistributionBalanceMeasuresMap.entries()].forEach(
+    ([measureName, measureInfo], i) => {
+      const measureValues = features.map(
+        (feature) => distributionBalanceMeasures[feature][measureInfo.KeyName]
+      );
+
+      // axis.left defines how far from the left to place the axis
+      // axis.width defines the width of the axis relative to the entire chart
+      xAxes.push({
+        categories: features,
+        left: hiddenCharts.has(measureName) ? "0%" : `${axisLeftStart}%`,
+        offset: 0,
+        showEmpty: false,
+        title: {
+          // If user hovers over the title, they are presented with the measure description.
+          text: `<div title="${measureInfo.Description}">${measureName} ${infoIcon}</div>`,
```

Comment:
This seems a little bit hacky, but I've faced similar issues with highcharts a few times. There's no way to do this with, say, a formatting function arg, is there?